client/src/views/viewers/AudioViewer.vue

```diff
@@ -114,6 +147,24 @@ function togglePlayback(): void {
   }
 }
 
+async function changePlaybackSpeed(value: number): Promise<void> {
+  const playbackSpeedValues = [0.25, 0.5, 1, 1.5, 2];
```

Comment:
We don't need an array, you could pass the value directly. And if we want to restrict the possible values, an enum would be more appropriate. Something like `await changePlaybackSpeed(PlaybackSpeed.Speed0_25)`.
I'm guessing you used an array because the zoom does, but for the zoom, a user can enter a value that has to be rounded, it's easier to do with an array. Here the values are fixed.